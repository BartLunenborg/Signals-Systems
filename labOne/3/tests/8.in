50 8 -0.78539 16 -2.35619
