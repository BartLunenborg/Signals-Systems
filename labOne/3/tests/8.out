x(t)=17.89*cos(2*pi*50.00*t-1.89)
